src/Extractors/Blade.php

```diff
@@ -13,10 +13,11 @@ class Blade extends Extractor implements ExtractorInterface
     /**
      * {@inheritDoc}
      */
-    public static function fromString($string, Translations $translations = null, $file = '')
+    protected static function fromStringDo($string, Translations $translations, $file)
     {
-        $string = (new BladeCompiler(new Filesystem(), null))->compileString($string);
+        $bladeCompiler = new BladeCompiler(new Filesystem(), null);
+        $string = $bladeCompiler->compileString($string);
```

Comment:
I splitted the constructor in its own line for PHP 5.3 compatibility. See http://3v4l.org/ZANaO vs http://3v4l.org/m9vk9